### ISSUE TITLE
2kan not finding installed game

### ISSUE THREAD

**Issue by jtac** [Loki-Lokster/2KAN#25]
default steam install position, but not finding game:
![image](https://user-images.githubusercontent.com/1128031/229044978-ffd7189e-5e1f-4902-b5e0-a7a5476233b3.png)

**Comment by LachlanR166** [comment]
Seems like something is not right with your setup, can you please upload the `2kan.txt` file found in the `2KAN > data > logs` directory.

**Comment by Loki-Lokster** [comment]
Seems like something is not right with your setup, can you please upload the `2kan.txt` file found in the `2KAN > data > logs` directory.

**Comment by jtac** [comment]
[2kan.log](https://github.com/KSP2-Toucan/TMM/files/11120988/2kan.log)
you mean the log?

**Comment by Loki-Lokster** [comment]
Yeah that's the one, did you extract the folder from the zip? The program doesn't seem to be getting past the initial setup which is odd, there's no errors indicated in the log file. Maybe try redownloading and trying a fresh extract and run.

**Comment by Loki-Lokster** [comment]
What system specs/operating system are you using? Do you use Microsoft Defender as your anti virus (if yes try running the `update_defender_definitions.bat` to update malware definitions)?

**Comment by BloodMisery** [comment]
I'm having the same issue. I download the file from your release page, zip the file into ksp2 folder, then run the program.
